mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -291,8 +292,14 @@ def list_model_endpoints(
         # Get the model endpoints records using sqlalchemy ORM
         with create_session(dsn=self._sql_connection_string) as session:
             # Generate the list query
-            query = session.query(self.model_endpoints_table).filter_by(
-                project=self.project
+            query = (
+                session.query(self.model_endpoints_table)
+                .options(
+                    # Exclude these fields when listing model endpoints to avoid returning too much data (ML-6594)
+                    sqlalchemy.orm.defer(EventFieldType.FEATURE_STATS),
+                    sqlalchemy.orm.defer(EventFieldType.CURRENT_STATS),
```

Comment:
```suggestion
                    mm_schemas.sqlalchemy.orm.defer(EventFieldType.FEATURE_STATS),
                    mm_schemas.sqlalchemy.orm.defer(EventFieldType.CURRENT_STATS),
```
- then you can also remove `from mlrun.common.schemas import EventFieldType`
- similar comment for the `kv_store`